package.json

```diff
@@ -17,46 +17,64 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each",
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
     "postpublish": "git push origin $(git branch | grep '*' | sed 's/* //'); git push origin --tags"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.18.0",
-    "ember-getowner-polyfill": "2.2.0"
+    "ember-cli-babel": "^7.26.6",
+    "ember-auto-import": "^2.4.2"
```

Comment:
Please don't include ember-auto-import v2 in this PR. Doing that means that anyone blocked by this repo needs to also upgrade to ember-auto-import v2 in order to get rid of the ember-cli-babel v6 issue.

Reply:
I'm pretty sure that I had to do that. I'll try rolling it back.

Reply:
`ember-source` depends on `ember-auto-import@2.4.2` so no, you can't avoid the upgrade

Reply:
No I remembered it wrong. Done it.